perses/app/setup_relative_calculation.py

```diff
@@ -53,6 +53,19 @@ def getSetupOptions(filename):
     if 'protocol-type' not in setup_options:
         setup_options['protocol-type'] = 'default'
 
+    if 'small_molecule_forcefield' not in setup_options:
+        setup_options['small_molecule_forcefield'] = 'gaff-2.11'
+
+    if 'small_molecule_parameters_cache' not in setup_options:
+        setup_options['small_molecule_parameters_cache'] = None
+
+    # Not sure why these are needed
+    # TODO: Revisit these?
```

Comment:
These are related to RJMC I believe, as to whether angle/dihedral terms go into the the jump, or into the alchemical protocol -- @dominicrufa will know better

Reply:
Thanks! I'll leave them for now!

In the future, I wonder if we could do something similar to YANK where each module has its own configuration options in a subsection of the YAML file, and can process its own `dict` representation of a subsection of the YAML file. That way, we don't need a monolithic method that knows about all possible options, but instead just dispatches `kwargs` to each submodule.

Reply:
Wait until you hear about experiments class